### PR TITLE
Implement auxpow merged-mining

### PR DESCRIPTION
**Pull request by reorder**

### CODE REVIEW

File: src/primitives/block.cpp [hunk 1]
```diff
@@ -9,6 +9,11 @@
 #include "tinyformat.h"
 #include "utilstrencodings.h"
 
+int GetOurChainID()
+{
+    return 0x56;
```
Comment (btcdrak) [inline src/primitives/block.cpp]: Maybe this should become a constant?

**Comment by btcdrak** [comment]
Well it's looking good to be honest. Tested on testnet seems to be mining well with non merged mined pools.

**Comment by reorder** [comment]
Please note mainnet nodes will have to -reindex too if running master after the merge and will produce blocks with auxpow version if used for mining.

**Comment by btcdrak** [comment]
That's a TODO for release notes/instructions.

**Comment by btcdrak** [comment]
tACK

**Pull request merged**